### DISCUSSION TITLE
Nginx中配置百度的UA能否实现加速

### DISCUSSION THREAD

**Discussion by fun-li** [AlistGo/alist#4640] [General]
看了文档介绍说配置pan.baidu.com的UA可以实现超级会员，想问问在Nginx请求中做一下统一配置是否可行

**Comment by fun-li** [comment]
少打了两个字，实现超级会员的加速

**Comment by anwen-anyi** [comment]
前提是你得有超级会员 有超级会员的前提下才会有后面的操作 否则一切白塔…

没试过 你这样的话那岂不是所有的都变成了pan.baidu.com?

自己尝试一下应该问题不大😁你可以自己试试看

**Reply by fun-li** [reply]
是的，就怕影响其他云盘的UA

**Comment by youtehub** [comment]
rclone 设置请求头，或者alist 最新版设置 UA
Ubuntu挂载示例：
```
 rclone mount baidu: /mountFolders  --vfs-cache-mode writes   --header 'Referer:https://pan.baidu.com/'  --header 'User-Agent:pan.baidu.com'
```
